src/screens/Inventory/PurchaseRequests.tsx

```diff
@@ -0,0 +1,59 @@
+import { createStyles, Theme, Typography, withStyles } from '@material-ui/core';
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { Link, RouteComponentProps } from 'react-router-dom';
+import BaseScreen from '../../components/BaseComponents/BaseScreen';
+import BaseScrollView from '../../components/BaseComponents/BaseScrollView';
+import Button from '../../components/Button';
+import { PurchaseRequestRecord } from '../../lib/airtable/interface';
+import { currentSitePurchaseRequestsSelector } from '../../lib/redux/inventoryData';
+import { PurchaseRequestStatus, SiteInventoryData } from '../../lib/redux/inventoryDataSlice';
+import PurchaseRequestCard from './components/PurchaseRequestCard';
+
+const styles = (theme: Theme) =>
+  createStyles({
+});
+
+interface PurchaseRequestsProps extends RouteComponentProps {
+  classes: {};
+  siteInventoryData: SiteInventoryData;
+}
+
+// TODO @wangannie: address empty state
+function PurchaseRequests (props: PurchaseRequestsProps) {
+    const { classes } = props;
+    const purchaseRequests = useSelector(currentSitePurchaseRequestsSelector);
+
+    const getPurchaseRequestsSection = (status: PurchaseRequestStatus) => (
+      <div>
+        <Typography variant="body1">{status}</Typography>
+        {purchaseRequests.filter(pr => pr.status == status).map((purchaseRequest: PurchaseRequestRecord) =>  (
+          <Link key={purchaseRequest.id} to={{ pathname: `/inventory/purchase-requests/purchase-request`, state: { purchaseRequest }}}>
```

Comment:
Are we fine passing purchaseRequest here via react-router? I'm good with it, just want to make sure it's intentional. A `currentPurchaseRequestId` might seem annoying to deal with

Reply:
Yeah I figured this flow would be fine (at least with the current use cases) since we don't really deal with editing a purchase request (besides reviewing, in which case the reviewer is sent back to the main list anyways). I'll probably re-evaluate once we have more finalized designs and user flows. If you think it's likely we're going to standardize  _everything_ to keeping an currentXXXId in state, I'm happy to refactor this too.